src/app/caesar-portal/common/components/pagination/pagination.component.ts

```diff
@@ -0,0 +1,46 @@
+import {Component, EventEmitter, Input, OnChanges, OnInit, Output, SimpleChanges} from '@angular/core';
+
+@Component({
+  selector: 'caesar-pagination',
+  templateUrl: './pagination.component.html',
+  styleUrls: ['./pagination.component.scss']
+})
+export class PaginationComponent implements OnInit, OnChanges {
+
+  constructor() { }
+
+  currentPage = 1;
```

Comment:
current page should be based on input, you may load a page already opened on 10th page and it should work properly